firmware/mcal/cli/periph/analog_output.h

```diff
@@ -0,0 +1,29 @@
+/// @author Tamara Xu
+/// @date 2024-10-03
+
+#pragma once
+
+#include <iostream>
+#include <string>
+
+#include "shared/periph/analog_output.h"
+
+namespace mcal::cli::periph {
+
+class AnalogOutput : public shared::periph::AnalogOutput {
+public:
+    AnalogOutput(std::string name) : name_(name) {}
+
+    void Set(float voltage) {
+        voltage_ = voltage;
+
+        std::cout << "Setting voltage " << name_ << " to " << voltage_ << " V" 
```

Comment:
This is a bit nit-picky, but can you make it print `Setting AnalogOutput "name_" to voltage V`? it'll align with our existing Digital input/output classes which print the class name